backend/backend/MemoryGameWithNumbers/Controllers/MemoryGameWithNumbersController.cs

```diff
@@ -23,12 +23,24 @@ public MemoryGameWithNumbersController(MemoryGameService memoryGameService)
     [HttpPost("attempt")]
     public ActionResult<bool> CheckAttempt([FromBody] List<int?> userAttempt)
     {
+        if (userAttempt.Count == 0)
```

Comment:
Cia may be a good idea just in case patikrint ar count'as yra in `[1; 16]`.

Reply:
Agreed